File: java/posledovatel.java
import java.util.Scanner;

public class posledovatel {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in, "UTF-8");
        System.out.print("Введите x:");
        double x = scanner.nextDouble();

        System.out.print("Введите y:");
        double y = scanner.nextDouble();

        double z = (x - y)/ Math.sqrt(x = y) + x * Math.pow(y, 2)/Math.sin(Math.pow(x, 2)) * Math.pow(Math.cos(y), 2);   
        System.out.println(z);
        scanner.close();
        }
}

File: Posledovatel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Posledovatel.cs
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Введите x = ");
            double x = Convert.ToDouble(Console.ReadLine());

            Console.WriteLine("Введите y = ");
            double y = Convert.ToDouble(Console.ReadLine());

            double z = (x - y) / Math.Sqrt(x + y) + x * Math.Pow(y, 2) / Math.Sin(Math.Pow(x, 2)) * Math.Pow(Math.Cos(y), 2);

            Console.WriteLine(z);

            Console.ReadLine();
        }
    }
}
